books/free-programming-books-fr.md

```diff
@@ -87,6 +87,9 @@
 * [Concevoir un Makefile](http://icps.u-strasbg.fr/people/loechner/public_html/enseignement/GL/make.pdf) - Vincent Loechner d'après Nicolas Zin (PDF)
 * [Introduction aux Makefile](http://eric.bachard.free.fr/UTBM_LO22/P07/C/Documentation/C/make/intro_makefile.pdf) (PDF)
 
+#### Mathématiques
+
+* [Approfondissements de lycée](https://fr.wikibooks.org/wiki/Approfondissements_de_lycée) - Zhuo Jia Dai (HTML)
 
```

Comment:
add another linebreak here. this will resolve the linter issue.

Reply:
another line break at line 93

Reply:
Added! Thank for your review.